packages/analytics-core/src/event-bridge/event-bridge-channel.ts

```diff
@@ -0,0 +1,31 @@
+import { Event, EventBridgeReceiver } from '@amplitude/analytics-types';
+
+export class EventBridgeChannel {
+  channel: string;
+  queue: Event[] = [];
+  receiver: EventBridgeReceiver | undefined;
+
+  constructor(channel: string) {
+    this.channel = channel;
+  }
+
+  sendEvent(event: Event) {
+    if (!this.receiver) {
+      this.queue.push(event);
+      return;
+    }
+    this.receiver.receive(this.channel, event);
+  }
+
+  setReceiver(receiver: EventBridgeReceiver) {
```

Comment:
I know the logic is `Only allow one receiver per channel`. But curious about how can we change the receiver?